proto/squad/farming/v1beta1/query.proto

```diff
@@ -196,6 +246,31 @@ key:
 };
 }
 
+// UnharvestedRewards returns unharvested rewards for a farmer
+rpc UnharvestedRewards(QueryUnharvestedRewardsRequest) returns (QueryUnharvestedRewardsResponse) {
```

Comment:
We need to update `docs/api/farming.md` and `docs/cli/farming.md` documents for this newly added query.

Reply:
Thanks, I update the doc.